Makefile

```diff
@@ -2,11 +2,18 @@ build:
 	@yarn turbo run build
 build-sequential:
 	@yarn turbo run --concurrency 1 build
+node-deps:
+	@yarn
+	build
 python-deps:
-	@python3 -m pip install -r images/plbase/python-requirements.txt --root-user-action=ignore
+	@python3.12 -m venv .venv
+	.venv/bin/python3 -m pip install --upgrade pip
+	.venv/bin/python3 -m pip install -r images/plbase/python-requirements.txt --root-user-action=ignore
```

Comment:
This fails for me on `pygraphviz`. I need to use the options recommended in the Graphviz install docs: https://github.com/pygraphviz/pygraphviz/blob/6cdd1eb676a67de697bf23d6948eabb525052eee/INSTALL.txt#L79-L82

Reply:
Ah, I have the following in my `~/.zshrc` which just makes this all work:

```sh
export LDFLAGS="-L/opt/homebrew/opt/graphviz/lib"
export CPPFLAGS="-I/opt/homebrew/opt/graphviz/include"
```

Is it worth documenting this?

Reply:
I think we should retain that bit of the native install docs from #11012

Reply:
To clarify, I think we should add something to the native install docs to specify that these lines should be added to `~/.zshrc` on MacOS when using Homebrew.